lib/gsl/oper.rb

```diff
@@ -40,4 +40,4 @@ def /(other)
 
 end
 
-[Fixnum, Float].each { |klass| klass.send(:include, GSL::Oper) }
+[RUBY_VERSION > "2.4" ? Integer : Fixnum, Float].each { |klass| klass.send(:include, GSL::Oper) }
```

Comment:
What if theres some outdated codebase that depends on Fixnum behaviour? IMO let this be for now we can release later versions with only Integer and Float.